react/legacy/FooterVtexLogo.js

```diff
@@ -52,7 +60,7 @@ FooterVtexLogo.propTypes = {
 }
 
 const getImagePathFromProps = ({ runtime, showInColor }) =>
-  `${isGoCommerceAccount(runtime.account) ? 'gocommerce' : 'vtex'}${
+  `${runtime.platform === PLATFORM_GOCOMMERCE ? 'gocommerce' : 'vtex'}${
```

Comment:
```suggestion
  `${runtime.platform}${
```

it should be enough, shouldn't it?

Reply:
If no other platform shows up, I think is fine 😅